styles.css

```diff
@@ -251,3 +280,193 @@ textarea {
   background-color: #2230d2;
   color: #fff;
 }
+
+@media (min-width: 600px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  #nav-links {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 28vw;
+    margin-right: 20px;
+  }
+
+  .logo {
+    margin-left: 53px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .intro {
+    display: flex;
+    flex-direction: column;
+    width: min-content;
+    margin: 10em 0 0 10em;
+  }
```

Comment:
- [ ] Please add some padding to these sections to better match the template.

![](https://i.imgur.com/DYuW5J4.png)